src/components/Header/AuxiliaryDropdown.tsx

```diff
@@ -32,23 +33,29 @@ export const AuxiliaryDropdown = () => {
       to: routes.hosts,
       onClick: () => sendEvent({ name: "Click All Hosts Link" }),
     },
+    {
+      text: "Task Queue",
+      to: getTaskQueueRoute(""),
```

Comment:
should we make the `distro` param optional (defaulting to `""`)  so you can just call this like `getTaskQueueRoute()`?